src/Tests/Microsoft.Diagnostics.Monitoring.UnitTests/BasicTests.cs

```diff
@@ -116,5 +116,15 @@ public async Task DisableMetricsViaCommandLineTest()
             Assert.Equal(HttpStatusCode.BadRequest, validationProblemDetailsException.StatusCode);
             Assert.Equal(StatusCodes.Status400BadRequest, validationProblemDetailsException.Details.Status);
         }
+
+        [Fact]
+        public async Task TestRequestLimits()
```

Comment:
Are you planning on filling this out (you don't have to at this time). If not, remove it.

Reply:
I can make this test since I still have to add the other routes to the client API.